<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/icons/tree_icon.png</normaloff>:/icons/icons/tree_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="MainTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="home_tab">
       <attribute name="title">
        <string>Home</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPieChart" name="chart" native="true">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>200</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="DateRangeWidget" name="date_range" native="true"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLegend" name="legend" native="true"/>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="rules_tab">
       <attribute name="title">
        <string>Rules</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QStatemenView" name="rulesView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="statements_tab">
       <attribute name="title">
        <string>Statements</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QStatemenView" name="statements_table"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="actionHome"/>
   <addaction name="actionImport_Bank_Statements"/>
   <addaction name="actionPythonConsole"/>
   <addaction name="actionAdd_rule"/>
   <addaction name="actionSet_category"/>
   <addaction name="actionChangeRule"/>
   <addaction name="actionDeleteRule"/>
  </widget>
  <action name="actionHome">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/go-home.png</normaloff>:/icons/icons/go-home.png</iconset>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
   <property name="toolTip">
    <string>Home</string>
   </property>
  </action>
  <action name="actionStatements">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/folder.png</normaloff>:/icons/icons/folder.png</iconset>
   </property>
   <property name="text">
    <string>Statements</string>
   </property>
   <property name="toolTip">
    <string>Statements</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/system-log-out.png</normaloff>:/icons/icons/system-log-out.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionPythonConsole">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/text-x-python.png</normaloff>:/icons/icons/text-x-python.png</iconset>
   </property>
   <property name="text">
    <string>PythonConsole</string>
   </property>
  </action>
  <action name="actionImport_Bank_Statements">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/document-import.png</normaloff>:/icons/icons/document-import.png</iconset>
   </property>
   <property name="text">
    <string>Import Bank Statements</string>
   </property>
  </action>
  <action name="actionAdd_rule">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/emblem-system.png</normaloff>:/icons/icons/emblem-system.png</iconset>
   </property>
   <property name="text">
    <string>Add rule</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="actionSet_category">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/go-jump.png</normaloff>:/icons/icons/go-jump.png</iconset>
   </property>
   <property name="text">
    <string>Set category</string>
   </property>
   <property name="toolTip">
    <string>Set category</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/media-floppy.png</normaloff>:/icons/icons/media-floppy.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionChangeRule">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/preferences-system.png</normaloff>:/icons/icons/preferences-system.png</iconset>
   </property>
   <property name="text">
    <string>Change rule</string>
   </property>
  </action>
  <action name="actionDeleteRule">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/edit-delete.png</normaloff>:/icons/icons/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete rule</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QPieChart</class>
   <extends>QWidget</extends>
   <header>qpiechart.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QLegend</class>
   <extends>QWidget</extends>
   <header>qlegend.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DateRangeWidget</class>
   <extends>QWidget</extends>
   <header location="global">daterangewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MainTabWidget</class>
   <extends>QTabWidget</extends>
   <header>maintabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QStatemenView</class>
   <extends>QTableView</extends>
   <header>qstatemenview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
